src/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -219,7 +219,15 @@ export default function CurrencyList({
   }, [currencies, otherListTokens])
 
   const Row = useCallback(
-    function TokenRow({ data, index, style }) {
+    function TokenRow({
+      data,
+      index,
+      style,
+    }: {
```

Comment:
generally like to define a separate interface `TokenRowProps`. also as i'm reading this code i have a feeling there is a cleaner way we could do this list, but we can leave it for another time